### PR TITLE
feat(batch-graph): special addresses

### LINKED ISSUE #505: Improve graph view
Not for this PR, and maybe not for now, but maybe we will want to: 

- Label some special accounts, like: `0xa03be496e67ec29bc62f01a428683d7f9c204930`
- Also, provide a way to open a node in etherscan or something. it's hard to check the actually full address

Nothing you should address here, just some thoughts

_Originally posted by @anxolin in https://github.com/cowprotocol/explorer/pull/504#pullrequestreview-1449661399_

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Closes #505 

Tagging special addresses

image

For now there are only 2, based on [this thread](https://cowservices.slack.com/archives/C0375NV72SC/p1685627041275649):
- 0xca771eda0c70aa7d053ab1b25004559b918fe662 -> CoW DAO
- 0xA03be496e67Ec29bC62F01a428683D7F9c204930 -> Solver Rewards Safe

Using a different icon for special nodes

# To Test

1. Load a regular batch
* The should be no changes
2. Load a "special" batch like `0x54450bc55a83276344a4e27b951a82d0d5e1927af0126aaac64b1417eb369868`
* The special icon and label should be used

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @alfetopito , great job!
One question: CoW Protocol Buffer icon is not clickable. Is this OK?
 
image

compare with the Solver Rewards Safe:
image

**Comment by alfetopito** [comment]
> Hey @alfetopito , great job! One question: CoW Protocol Buffer icon is not clickable. Is this OK?
> 
> image
> 
> compare with the Solver Rewards Safe: image

Hmm yeah, as it's the exact same address as the protocol itself.
Still, will see if I can quickly add it

File: src/api/tenderly/tenderlyApi.ts [hunk 1]
```diff
@@ -186,3 +191,11 @@ function _contractName(name: string): string {
...
   return name
 }
+
+export function getAliasFromAddress(address: string, isUnknown = false): string {
+  const lowerCaseAddress = address.toLowerCase()
+
+  if (SPECIAL_ADDRESSES[lowerCaseAddress]) return SPECIAL_ADDRESSES[lowerCaseAddress]
+
+  return isUnknown ? abbreviateString(address, 6, 4) : ALIAS_TRADER_NAME
+}
```
Comment (anxolin) [inline src/api/tenderly/tenderlyApi.ts:201]: i don\t think this is related to tenderly, consider moving to a more central point

i.e. We might want to use it for resolving it also in the lists

**Review by anxolin — APPROVED** [review]
Soft approve

**Review by shoom3301 — APPROVED** [review]

**Pull request merged**